src/components/common/form/__tests__/__util__/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { Provider as PaperProvider } from 'react-native-paper'
+import { theme as defaultTheme } from '../../../../theme/styles'
+
```

Comment:
I think we should move this utils file to a root folder, or a generic place, because we will use it from most of the components, at least common components

Reply:
maybe using webpack aliases to import 'theme'?